edx_rest_framework_extensions/decorators.py

```diff
@@ -2,8 +2,8 @@
 from functools import wraps
 
 import waffle
-from edx_rest_framework_extensions.authentication import JwtAuthentication
 from edx_rest_framework_extensions.config import SWITCH_ENFORCE_JWT_SCOPES
+from edx_rest_framework_extensions.authentication import is_jwt_authenticated
 
 
 def skip_unless_jwt_authenticated(f):
```

Comment:
Should we get rid of this if the plan is to use `rest_condition`?

Reply:
I was just thinking that if users of the permission classes decided to do something like:
```
permission_classes = [And(IsJwtAuthenticated, JwtHasScope)]
```
then we would be running the `is_jwt_authenticated` logic twice, but that probably isn't an issue.